pkg/daemon/pivot/types/imageinspection.go

```diff
@@ -3,4 +3,9 @@ package types
 const (
 	// PivotNamePrefix is literally the name prefix of the new pivot
 	PivotNamePrefix = "ostree-container-pivot-"
+
+	// PivotFailurePath is the path pivot writes its error to when it fails. We need this to bubble
+	// up any error happning there as we just log those errors in MCD and are useful unless we report
+	// them up the stack
+	PivotFailurePath = "/etc/pivot/failure"
```

Comment:
Let's store things like this in `/run` - there's no reason to persist across reboots.

Reply:
or go with

> take all lines from the journal output that start with error or so

I'm ok either way